src/types/projects.ts

```diff
@@ -187,30 +187,29 @@ export interface JobDefinition {
 }
 
 export interface ListProject {
-  id: string
-  ext_id: string
-  reference_number: string
-  institution_id: string
-  type_classifier_value_id: string
+  id?: string
+  ext_id?: string
+  reference_number?: string
+  institution_id?: string
+  type_classifier_value_id?: string
   type_classifier_value?: TypeClassifierValue
-  comments: string
-  workflow_template_id: WorkflowTemplateID
-  workflow_instance_ref: string | null
-  deadline_at: string
-  created_at: string
-  updated_at: string
+  comments?: string
+  workflow_template_id?: WorkflowTemplateID
+  workflow_instance_ref?: string | null
+  deadline_at?: string
+  created_at?: string
+  updated_at?: string
   sub_projects: ListSubProjectDetail[]
-  status: ProjectStatus
-  tags: Tag[]
-  price: string
+  status?: ProjectStatus
+  tags?: Tag[]
+  price?: string
   event_start_at?: string
-}
-
-export interface ProjectDetail extends ListProject {
```

Comment:
**[blocker]**

Let's stick with ProjectDetail, the type was renamed to ProjectDetail on purpose, so it would be more consistent (We have SubProjectDetail, so ProjectDetail is more intuitive that DetailedProject). Basically we want naming to be intuitive and predictable, so every type that has a "Detail" and "List" version of it, we should keep the consistency. In this case it will be "ObjectNameDetail" instead of "DetailedObjectName". We can use the "DetailedObjectName" as well, but then it should be changed for every object

Reply:
Actually when we consider the naming structure for ListProject and ListSubProjectDetail, then better naming would be to have the object name in the end.

So ideally: 

1. ListSubProject
2. DetailedSubProject
3. ListProject
4. DetailedProject

But yeah not too important right now, we can deal with naming consistency a bit later